_posts/es/pages/2017-01-01-download.md

```diff
@@ -83,29 +83,29 @@ checksum_warning_and_ok: 'En la salida producida por el comando superior, puedes
 
 example_builders_line: "E777299FC265DD04793070EB944D35F9AC3DB76A Michael Ford (fanquake)"
 builder_keys_url: "https://github.com/bitcoin/bitcoin/tree/master/contrib/builder-keys"
+example_builder_key_file: "fanquake.gpg"
 
 obtain_release_key: >
-  <p>Las versiones de Bitcoin están firmadas por varias personas, cada una de ellas con
-  una clave pública única. Para reconocer la validez de las firmas, debes usar GPG para
-  cargar estas claves públicas localmente. Puedes encontrar muchas claves de
-  desarrolladores listadas en el <a
-  href='$(BUILDER_KEYS_URL)'>bitcoin/bitcoin repository</a>, las cuales puedes entonces
-  cargar en tu base de datos de claves GPG.</p>
-
-  <p>Por ejemplo, dada la línea <a href='$(BUILDER_KEYS_TXT_URL)'><code>
-  builder-keys/keys.txt</code></a>
-  <code>$(EXAMPLE_BUILDERS_LINE)</code> podrías cargar esa clave usando este
-  comando:</p>
+  <p>Las publicaciones de código son firmadas por varias personas, cada una de ellas
+  identificada por una única clave pública. Para poder validar las firmas, debes utilizar
+  GPG para cargar de forma local estas claves públicas. Encontrarás varias de las clases de
+  diversos desarroladores en <a href='$(BUILDER_KEYS_URL)'>el repositorio bitcoin-core/guix.sigs.</a>
+  Puedes utilizar esta lista para cargar las claves en tu base de datos de GPG.</p>
+
+  <p>Por ejemplo, puedes cargar la clave
+  <a href='$(BUILDER_KEYS_URL)/$(EXAMPLE_BUILDER_KEY_FILE)'><code>builder-keys/$(EXAMPLE_BUILDER_KEY_FILE)</code></a>
+  descargando el archivo <code>$(EXAMPLE_BUILDER_KEY_FILE)</code>
+  y utilizando el siguiente comando:</p>
 
 choosing_builders: >
-  Sería recomendable que eligieras varias personas de esta lista que estimes
-  confiables e importes sus claves según se indica arriba, o importes todas las
-  claves según las instrucciones en el documento <a
-  href="$(BUILDER_KEYS_URL)"><code>contrib/builder-key</code>
-  README</a>. Después usarás sus claves para comprobar las firmas que atestiguan la
-  validez de las checksums que usas para comprobar los programas binarios.
+  Es recomendable que elijas a varias de las personas de esta lista que consideres de
+  confianza e importes sus claves utilizando el método previamente explicado. Más
+  adelante, utilizarás sus claves para validar las firmas que atestan la validez
+  de los checksums que se proporcionan conjuntamente con los binarios del código.
```

Comment:
For consistency with the rest of the text, this should read "las checksums" instead of "los checksums"